### ISSUE TITLE
Deprecated parameter in get_the_excerpt

### ISSUE THREAD

**Issue by rhurling** [jesperbjerke/wa-fronted#13]
`get_the_excerpt($post_id)` is used here https://github.com/jesperbjerke/wa-fronted/blob/master/wa-fronted.php#L569, but the parameter is deprecated and will only return the excerpt of the post currently in the Loop, which could possibly override some data, especially since you perform a saving function with that data later.
You could instead use something like this:
```php
$this_autosave_data['post_excerpt'] = apply_filters( 'get_the_excerpt', get_post_field('post_excerpt', $post_id, 'raw'));
```
This would do the same thing, only with data from the correct post instead of an empty string (since ajax actions by default shouldn't be in a Loop and I didn't look whether you initialise one yourself).

See:
https://developer.wordpress.org/reference/functions/get_the_excerpt/
https://codex.wordpress.org/Function_Reference/get_the_excerpt

**Comment by jesperbjerke** [comment]
Thanks! Will fix. :)

**Comment by jesperbjerke** [comment]
fixed in v1.1
